packages/cli/src/commands/migrate/index.ts

```diff
@@ -38,6 +45,7 @@ migrateCommand
   .option('-i, --interactive', 'interactive mode')
   .option('-v, --verbose', 'print debugging logs')
   .option('-d, --dryRun', 'print files that will be attempted to migrate', false)
+  .option('-r, --regen', 'print out current migration status')
```

Comment:
Thinking of naming on this.

Does it make more sense to name this `--rerun`.

This use case (to my understanding) is that I've run `rehearsal migrate` and now I am wanting to regen a report of errors?

What is the by-product of this argu `--regen` does it only create a new sarif report?

Just thinking through this, it's more of a `--status` and it updates the report?

Reply:
@wondersloth In my current implementation, the difference between `migrate --regen` and `migrate` is that `migrate` also includes autofixes, and `migrate --regen` does not. `migrate --regen` will only mark the ts errors with comments, and print out ts errors and eslint errors in report. But I do see your point. What you do think, @lynchbomb ?

Reply:
(Lets keep `regen`. This isn't technically a rerun since it doesn't actually migrate/upgrade anything. We only want to reproduce the report.